tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -996,8 +996,8 @@ def parse_layouts_by_destination(s: str, parsed: dict, dest: str = None) -> None
             elif m2:
                 found_g = m2.groups()
             else:
-                raise Error("More then one layout provided for --{}layout without providing name.".format(
-                    dest + '_' if dest else ''))
+                raise Error("Invalid usage of --layout parameter. Please use following syntax:\n name(nchw)\n "
```

Comment:
Can you provide sample of how this will look? It seems a bit ugly to me. Maybe consider using more spaces or other symbols like * to make it look better. But generally, change is good.

Reply:
[ ERROR ]  Invalid usage of --layout parameter. Please use following syntax for each tensor or opeartion name:
name(nchw)
name(nhwc->nchw)
name[nchw]
name[nhwc->nchw]

Not sure how to say more generic. What do you think?

Reply:
BTW, name[nhwc->nchw] is not a correct syntax

Reply:
I must have misunderstood "name1(nhwc->[n,c,h,w])" case.

Reply:
```
[ ERROR ] Invalid usage of --{}layout parameter. Please use following syntax for each tensor or operation name:
  name(nchw)
  name(nhwc->[n,h,w,c])
  name[n,c,h,w]
  name[n,h,w,c]->[n,c,h,w]
```
I suggest something like this, please add at least 2 spaces before each syntax suggestion. Also please mind the typo in **щзукфешщт**. And also leave the formatting for `--{}layout`, because this can be not only `--layout`, but also `--target_layout` and `--source_layout`.

Reply:
Done.